tests/topotests/lib/common_config.py

```diff
@@ -2971,6 +2971,8 @@ def addKernelRoute(
             ip, mask = grp_addr.split("/")
             if mask == "32" or mask == "128":
                 grp_addr = ip
+        else:
+            mask = 32
```

Comment:
This works, but the int/str difference along the 2 paths and with the comparison, and the v4 centric value is sort of itchy. What about setting it to "not 0" :) Or perhaps less humorously
```
    mask = "32" if addr_type == "ipv4" else "128"
```

Reply:
chagned to this